Subsonicious/Subsonicious/Views/RootView.swift

```diff
@@ -27,7 +27,11 @@ struct RootView: View {
             }
         }
         .onAppear {
-            isLoggedIn = (try? authenticationManager.authenticateWithPersistedServer()) == true
+            do {
+                isLoggedIn = try authenticationManager.authenticateWithPersistedServer()
+            } catch {
+                debugPrint(error)
```

Comment:
maybe you need to set isLoggedIn too in that case as before ? I don't know

Reply:
I don't think so, because it already have a default value